### PR TITLE
feature: Add docker creation

### PR DESCRIPTION
**Pull request by laurentheirendt**
this should work 😄 

repo needs to be bumped and tagged

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	mkdir "$JULIA_PATH"; \
+	tar -xzf julia.tar.gz -C "$JULIA_PATH" --strip-components 1; \
+	rm julia.tar.gz; \
+	\
+	apt-mark auto '.*' > /dev/null; \
+	[ -z "$savedAptMark" ] || apt-mark manual $savedAptMark; \
+	apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false; \
+	\
+# smoke test
+	julia --version; \
```
Comment (exaexa) [inline Dockerfile]: Is there any difference between doing all this manually and just installing julia from 1] debian repo 2] starting FROM the julia images here https://hub.docker.com/_/julia/ ?

**Review by exaexa — COMMENTED** [review]
note: it might be useful to squash the commits

File: Dockerfile (hunk 1 above)
Comment (laurentheirendt) [reply Dockerfile]: I could try that ...

**Review by exaexa — APPROVED** [review]
LOVELY.

**Pull request merged**